src/CSET/recipes/simple_mask.yaml

```diff
@@ -0,0 +1,36 @@
+category: Quick Look
+title: Example simple mask application
```

Comment:
Minor point, is it worth having the filename start with example_ for this example yaml files?

Reply:
Renamed.